analysis/wwz/wwz4l.py

```diff
@@ -584,7 +631,10 @@ def process(self, events):
             ######### Masks we need for the selection ##########
 
             # Pass trigger mask
```

Comment:
If I understand properly, the era is not wanted for this function for R2, but it's not None. Instead of putting the whole call in an if statement (which is hard to read), I think it would be more clear to just use a statement to set a bool and then only have one function call as before. I.e. add this:
```
era_for_trg_check = era
if not (is2022 or is2023):
    era_for_trg_check = None
```
Then just call the `es_tc.trg_pass_no_overlap()` once and pass `era_for_trg_check` to it for era.

Reply:
done

Reply:
See comment above where you changed how we define eras. I think it's a lot easier to read if we leave that alone and just directly add this `era_for_trg_check` statement here directly above the trigger check line.

Reply:
done